### ISSUE TITLE
WordPress 5.6 Breaks the Submission Previews

### ISSUE THREAD

**Issue by cogdog** [cogdog/ds106bank#17]
I've definitely confirmed that something in WP 5.6 stops the fancybox (this theme has used a deprecated version of fancybox, 2.1.5( previews for submitting new assignments or responses. No errors are reported, but the action never fires, and thus users can never submit items because the preview never generated.

After failing.fiddling with efforts to use the newest 3.x version of fancybox (it's completely rewritten) and trying to use built in WordPress versions of Thickbox (it's ugly and not responsive), my plan is to redo the way previews are done to be in line with other splots (generating a preview in a new window, via WordPress) that will thus preview more close to the way it will appear when published, plus like more web accessible than an overlay of content.

If this is stopping your use of the site, the best suggestion I can do is see if your host can roll back your version of WordPress to before 5.6

**Comment by woodwardtw** [comment]
One option that seems to work is to install the [JQuery Migrate](https://wordpress.org/plugins/enable-jquery-migrate-helper/) plugin and then go to Tools>jQuery Migrate and set the jQuery Version to 1.12.4-wp.

**Comment by cogdog** [comment]
Thanks that's damn clever. I know it messaged things up for a Middlebury site. But I hear they have a top notch programmer on hand.

I really need to redesign that flow there, the lightbox for previews is a bit duct taped, and my other SPLOTs I have rigged a way to use native WordPress previews. Just need to prey some time to redo my old sloppy code.

**Comment by woodwardtw** [comment]
Yep that's what I was messing with today. :) Normally the jquery migrate plugin gives some decent console messages about what's up but nada for this so I resorted to my typical measures. I tried to be more helpful but failed.
